Implementation/FindMyBLEDevice/FindMyBLEDevice/FindMyBLEDevice.csproj

```diff
@@ -5,12 +5,17 @@
     <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugType>full</DebugType>
+    <DefineConstants>TRACE;DEBUG;NETSTANDARD;NETSTANDARD2_0;ENABLE_TEST_CLOUD;</DefineConstants>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+  </PropertyGroup>
```

Comment:
Is this change necessary / sensible?

Reply:
Yes. It is a necessary step for UI testing on iOS, as per official docs